### ISSUE TITLE
Clean build of commit 6a264a2, schema Single not existed in parent catalog

### ISSUE THREAD

**Issue by ingted** [NTDLS/NTDLS.Katzebase.Server#73]
![image](https://github.com/user-attachments/assets/036b04b9-531c-4bce-904f-ee2bb6722394)

**Comment by ingted** [comment]
Reproduce script:

```
#if INTERACTIVE
#r @"nuget: Newtonsoft.Json, 13.0.3"
#r @"nuget: NTDLS.DelegateThreadPooling, 1.4.8"
#r @"nuget: NTDLS.FastMemoryCache, 1.7.5"
#r @"nuget: NTDLS.Katzebase.Client, 1.7.8"
#r @"nuget: Serilog, 4.0.1"
#r @"nuget: NTDLS.Helpers, 1.2.9.0"
#r @"nuget: NTDLS.ReliableMessaging, 1.10.9.0"
#r @"../../../NTDLS.Katzebase.Shared/bin/Debug/net8.0/NTDLS.Katzebase.Shared.dll"
#r @"../../../NTDLS.Katzebase.Engine/bin/Debug/net8.0/NTDLS.Katzebase.Engine.dll"
#endif
open Newtonsoft.Json
open NTDLS.Katzebase.Shared
open NTDLS.Katzebase.Engine
open System.IO
open System.Reflection

let json = 
    try
        File.ReadAllText("appsettings.json");
    with
    | _ ->
        let fi = new FileInfo(Assembly.GetEntryAssembly().Location)
        File.ReadAllText(fi.DirectoryName + @"\appsettings.json")


let settings = JsonConvert.DeserializeObject<KatzebaseSettings>(json)

let createIfDirNotExisted p =
    let di = DirectoryInfo p
    if not di.Exists then
        di.Create()

createIfDirNotExisted settings.DataRootPath
createIfDirNotExisted settings.TransactionDataPath
createIfDirNotExisted settings.LogDirectory

//prevent missing single schema for ver.0.0.1
createIfDirNotExisted $"{settings.DataRootPath}/single"

let _core = new EngineCore(settings)
```

**Comment by ingted** [comment]
Since I am developing the unit test project, so in my local engine repo, I need to add stuff like this to bypass the error so far. 

```
#if UNITTEST_VER_0_0_1_MOD
            if (false)
#endif
            Documents.InsertDocument(systemSession.Transaction, "Single", "{ephemeral: null}");
```

**Comment by ingted** [comment]
Seems resolved around Commit d293eba5237f87573a0c5722ab2fc9f6f19f0828

**Comment by NTDLS** [comment]
Yea, I really butchered the creation of "Temporary" and "Single" a few commits ago, this should be fixed up quite nicely now.
